code/addons/actions/package.json

```diff
@@ -77,8 +77,8 @@
     "!src/**/*"
   ],
   "scripts": {
-    "check": "../../../scripts/prepare/check.ts",
-    "prep": "../../../scripts/prepare/bundle.ts"
+    "check": "node --loader esbuild-register/loader -r esbuild-register ../../../scripts/prepare/check.ts",
```

Comment:
I see that this is what is [documented in `esbuild-register`](https://github.com/egoist/esbuild-register#experimental-loader-support), but based on the Node.js docs for v18 it looks like `--loader` is still `--experimental-loader`, so I'd expect this to always be ignored and just fallback to the `--require/-r` flag instead

I wonder if this is more correct/faster/makes any difference:

```
node --experimental-loader esbuild-register/loader --require esbuild-register ../../../scripts/prepare/check.ts
```

Reply:
If `--loader` would be ignored from Node, any attempt resolving esm-related imports would fail, wouldn't it?

Reply:
I _think_ that's what the `-r esbuild-register` part does instead, but I'm on deep water here.